src/overlays/actors/ovl_Obj_Dora/z_obj_dora.c

```diff
@@ -98,34 +152,188 @@ static DamageTable D_80B61488 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
```

Comment:
You can make and use an enum for these,
```c
typedef enum {
    /* 0x0 */ GONG_DMGEFF_NONE,
    /* 0xE */ GONG_DMGEFF_STRONG = 0xE,
    /* 0xF */ GONG_DMGEFF_LIGHT
} ObjDoraDamageEffect;
```
I suppose, given how the switch works: you can then use them in the switch in `CheckCollision`.